decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/application/ValidationService.java

```diff
@@ -52,7 +52,7 @@ public static void validateHeaderFrom(BusinessAppHdrV02 header, String truststor
         String domain = header.getFr().getFIId().getFinInstnId().getNm();
 
         if (truststorePath != null && pwd != null) {
-            checkSSLCertificate("https://" + domain, truststorePath, pwd);
+            checkSSLCertificate("https://" + domain + ":8443", truststorePath, pwd);
```

Comment:
The port should probably be defined in a .properties file.

Reply:
I added the port as an argument